recipe/meta.yaml

```diff
@@ -76,6 +79,20 @@ test:
     - test -f ${PREFIX}/include/{{ each_ncurses_header }}
     {% endfor %}
 
+    # Test for tinfo symbols.
+    {% set tinfo_symbols = [
+        "cbreak",
+        "keypad",
+        "stdscr"
+    ] %}
+
+    {% for each_tinfo_symbol in tinfo_symbols %}
+    - nm "${PREFIX}/lib/libncurses.so" | grep '.* {{ each_tinfo_symbol }}$'       # [linux]
+    - nm "${PREFIX}/lib/libncursesw.so" | grep '.* {{ each_tinfo_symbol }}$'      # [linux]
+    - nm "${PREFIX}/lib/libncurses.dylib" | grep '.* _{{ each_tinfo_symbol }}$'   # [osx]
+    - nm "${PREFIX}/lib/libncursesw.dylib" | grep '.* _{{ each_tinfo_symbol }}$'  # [osx]
+    {% endfor %}
```

Comment:
Check for a few symbols that would be found in `libtinfo`, but will now be in `libncurses`. See this issue ( https://github.com/conda-forge/ncurses-feedstock/issues/9 ) for details.